### PR TITLE
fix: Rollback admin/workspace route to admin/info

### PR DESCRIPTION
**Pull request by hugocostadev**
## Proposed changes (including videos or screenshots)

Cherry peak of this merged PR: https://github.com/RocketChat/Rocket.Chat/pull/30648

### CODE REVIEW

**Review by d-gubert — CHANGES_REQUESTED** [review]
Please include a changeset file explaining what has been fixed https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#4.-adding-changeset-to-your-pull-request

File: .changeset/quick-cups-sparkle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Change `/admin/workspace` path to `/admin/info` path as it was before
```
Comment (d-gubert) [inline .changeset/quick-cups-sparkle.md:5]:
```suggestion
Rolled back a change of route from `/admin/workspace` to `/admin/info`
```

Suggestion so the message aligns with the guidelines for changesets https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#4.-adding-changeset-to-your-pull-request

**Review by d-gubert — CHANGES_REQUESTED** [review]

File: .changeset/quick-cups-sparkle.md (hunk 1 above)
Comment (hugocostadev) [reply .changeset/quick-cups-sparkle.md:5]: I don't this that Rolled back it's a good term to use, since it's a technical term

Comment (d-gubert) [reply .changeset/quick-cups-sparkle.md]: That's a good point, please feel free to suggest something different

**Pull request merged**